docs/web-console-guide/master.adoc

```diff
@@ -33,7 +33,7 @@ The {ProductName} ({ProductShortName}) {WebName} has two views:
 * Administration view
 * Migration view
 
-In *Administration* view, you configure the instance environment, working with credentials, repositories, and HTTP and HTTPS proxy definitions.
+In *Administration* view, you configure the instance environment, working with credentials, repositories, HTTP and HTTPS proxy definitions, and issue management.
 
```

Comment:
Maybe add "custom migration targets"  after "proxy definitions" ? Although makes a long sentence even longer, it makes it clear that "HTTP and HTTPS proxy definitions" is a single unit, at least to my ear.